images/varnish-drupal/drupal.vcl

```diff
@@ -279,7 +279,7 @@ sub vcl_hit {
       return (deliver);
     } else {
       # No candidate for grace. Fetch a fresh object.
-      return (miss);
+      return (pass);
```

Comment:
@seanhamlin - apparently `miss` never worked - is pass appropriate here?

OR should we implement the recommended steps from here - https://varnish-cache.org/docs/6.2/whats-new/upgrading-6.2.html#whatsnew-upgrading-vcl-2019-03

Reply:
Yeah, the docs seem clear on what is needed to reproduce this behaviour in 6.2, with the `return (restart)`.

Reply:
I'll have another look at the `vcl_recv{}` section to incorporate the code - I'll reach out for vcl ninja skills...